geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitorJUnitTest.java

```diff
@@ -268,12 +266,12 @@ public void testSuspectMembersCalledThroughMemberCheckThread() throws Exception
 
     // when the view is installed we start a heartbeat timeout. After
     // that expires we request a heartbeat
-    Thread.sleep(3 * memberTimeout + 100);
+    Awaitility.await().atMost(3 * memberTimeout + 100, TimeUnit.MILLISECONDS)
+        .until(() -> (gmsHealthMonitor.isSuspectMember(mockMembers.get(myAddressIndex + 1))) &&
```

Comment:
You could use "untilAsserted" here. You would have to use a soft assertion if you wanted to see all three of those values every time, but even if you just use regular assertions you'll get an error message that is a little more informative (as it stands, you'll just see "expected true and was false". until asserted would tell you which assertion failed and with what value)

Reply:
Testing this it looks like awaitility swallows the assertion failure and simply outputs: ```org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a lambda expression in org.apache.geode.distributed.internal.membership.gms.fd.GMSHealthMonitorJUnitTest null within 1000 milliseconds.```

Since this isn't really an improvement, I'm going to leave the existing code.